wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/FieldExtensions.kt

```diff
@@ -0,0 +1,25 @@
+/**
+ * Extension functions/properties for the Field class
+ */
+package com.squareup.wire.kotlin
+
+import com.squareup.wire.schema.Field
+import com.squareup.wire.schema.ProtoType
+
```

Comment:
Neat! I like the extensions that fold a level of nesting, e.g. `type.valueType() -> valueType`, but I think introducing property extensions just to turn a method into a property is a bit of an overkill. Plus, we might kotlinize the schema module in future and will turn those into properties anyway.

Reply:
I see what you mean about it being a bit overkill. I feel like it can make code a bit more natural to work with, however (properties are data that the object owns, functions do some action). That's at least what I was thinking when I wrote this. But it is extra code to read that doesn't do very much. I can remove them.

What did you think about the other uses of properties where it isn't just replacing a function with the same name?

Reply:
Yeah, I like those. Let's keep the other ones for consistency. If we decide to rewrite the schema module in Kotlin we can use these extensions. Just one thing - please make these extensions `internal`, we don't want them in the API.